src/core/include/openvino/op/util/sub_graph_base.hpp

```diff
@@ -18,10 +18,10 @@ class OPENVINO_API SubGraphOp : public MultiSubGraphOp {
     OPENVINO_OP("SubGraphOp", "util", op::util::MultiSubGraphOp);
     BWDCMP_RTTI_DECLARATION;
 
-    virtual const std::shared_ptr<Function>& get_function() const {
+    virtual const std::shared_ptr<Model>& get_function() const {
         return m_bodies[0];
     };
-    virtual void set_function(const std::shared_ptr<Function>& func) {
+    virtual void set_function(const std::shared_ptr<Model>& func) {
```

Comment:
`set_function` => `set_subgraph`?

Reply:
set_model? :)

Reply:
here the model is used to denote subgraph..

Reply:
I cannot rename this method right now, because it breaks the backward compatibility for TensorIterator: https://github.com/openvinotoolkit/openvino/blob/releases/2021/4/ngraph/core/include/ngraph/op/tensor_iterator.hpp#L39

Reply:
and what to do with it?

Reply:
We can deprecate old method and introduce new.